src/osx/cocoa/utils_base.mm

```diff
@@ -36,6 +36,8 @@
 #if (defined(__APPLE__) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 101000) \
     || (defined(__WXOSX_IPHONE__) && defined(__IPHONE_8_0))
     #define wxHAS_NSPROCESSINFO 1
+#else
+    #define wxHAS_NSPROCESSINFO 0
```

Comment:
By convention, `wxHAS_XXX` are supposed to be undefined instead of being defined as 0 (unlike `wxUSE_XXX` which should be always defined as 1 or 0).